lib/bgs/helpers/formatters.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
```

Comment:
"Helper" usually means a code smell. All of the logic in here looks like serialization logic. I think if we had some better encapsulation around the BGS response object models, we might then also be able to encapsulate most/all of this serialization logic inside either the model or a dedicated serializer for the model.